lib/pdk/validators/metadata/metadata_syntax.rb

```diff
@@ -42,7 +42,10 @@ def self.stop_spinner(exit_code)
       end
 
       def self.invoke(report, options = {})
-        targets, _skipped, _invalid = parse_targets(options)
+        targets, skipped, invalid = parse_targets(options)
+
+        process_skipped(skipped)
+        process_invalid(invalid)
```

Comment:
I think we need to pass the report object here. `process_skipped(report, skipped)` and `process_invalid(report, invalid)`